CMakeLists.txt

```diff
@@ -16,11 +16,64 @@ endif()
 find_package(ament_cmake REQUIRED)
 find_package(rosidl_default_generators REQUIRED)
 
+set(protobuf_MODULE_COMPATIBLE TRUE)
+find_package(Protobuf CONFIG REQUIRED)
+message(STATUS "Using protobuf ${Protobuf_VERSION}")
+
+set(_PROTOBUF_LIBPROTOBUF protobuf::libprotobuf)
+set(_REFLECTION gRPC::grpc++_reflection)
+
+find_package(gRPC CONFIG REQUIRED)
+message(STATUS "Using gRPC ${gRPC_VERSION}")
+
+# Generated sources
+add_custom_command(
+      OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/akushon.pb.cc" 
+            "${CMAKE_CURRENT_BINARY_DIR}/akushon.pb.h"
+            "${CMAKE_CURRENT_BINARY_DIR}/akushon.grpc.pb.cc"
+            "${CMAKE_CURRENT_BINARY_DIR}/akushon.grpc.pb.h"
+      COMMAND $<TARGET_FILE:protobuf::protoc>
+      ARGS --grpc_out "${CMAKE_CURRENT_BINARY_DIR}"
+        --cpp_out "${CMAKE_CURRENT_BINARY_DIR}"
+        -I ${CMAKE_CURRENT_SOURCE_DIR}/proto
```

Comment:
Is it possible to remove this line?

It seems that setting include here is unused because `akushon.proto` is the only file in the `${CMAKE_CURRENT_SOURCE_DIR}/proto`.

```suggestion
```

Reply:
no mas, it still has to be included even if there's only 1 file in this directory
`
File does not reside within any path specified using --proto_path (or -I).  You must specify a --proto_path which encompasses this file.
`

Reply:
Aah i see. Got it.